src/assets/contributors-map.geojson

```diff
@@ -70,11 +58,20 @@
       },
       "geometry": {
         "type": "Point",
-        "coordinates": [
-          -118.2422725993683,
-          34.05295703301907
-        ]
+        "coordinates": [-118.2422725993683, 34.05295703301907]
+      }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "firstName": "Joey",
+        "githubId": "marinojoey",
+        "favoriteCrag": "8af5df63-9dd1-5fb2-9ef1-c3c3860494f5"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [39.737812, -105.324479]
```

Comment:
Can you swap the order to [-105.324479, 39.737812]?  If coordinates are in the right order (longitude, latitude) GitHub will render the data on a map.